src/common/paddle/index.ts

```diff
@@ -98,9 +97,15 @@ export const getProductPurchaseType = ({
     throw new Error('Core product id is not set');
   }
 
+  if (!remoteConfig.vars.plusOrganizationProductId) {
+    throw new Error('Plus organization product id is not set');
+  }
```

Comment:
Just need to make sure it's set when we merge else it will also break other paddle hooks?

Reply:
You have already set it I think?